Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Can you remove this file from the pull request

    git rm Testing/Temporary/CTestCostData.txt

and add the following line to the end of the .gitignore file

    Testing/*